src/functional/iterator/transforms/unroll_reduce.py

```diff
@@ -110,7 +81,9 @@ def visit_FunCall(self, node: ir.FunCall, **kwargs):
 
         offset_provider = kwargs["offset_provider"]
         assert offset_provider is not None
-        connectivity = _get_connectivity(node.args, offset_provider)
+        connectivity = reduction_utils.get_connectivity(
+            node, offset_provider  # type: ignore[arg-type]
```

Comment:
I get `FunCall` is not `_FunCall`. Can I resolve that, or is that the problem that you mentioned @egparedes ?

Reply:
Maybe I'm confused, but the error I see in the failing test log is: `'FunCall' object is not iterable`, and I think it's caused by simple mistake in `test_reduction.utils:test_get_partial_offsets`, where I think `_get_partial_offset_tag()` should be called instead of `test_get_partial_offsets()`

Reply:
The failure is unrelated to this issue (the failure was because of a change I made in the get_partial_offsets, taking funcall.args not the reduce funcall itself). `get_connectivity` should take a `_FunCall`, but mypy doesn't understand that `itir.FunCall` is a subclass of `_FunCall`.